samples/gen-js-no-definitions/index.js

```diff
@@ -0,0 +1,105 @@
+
+const discovery = require("@clever/discovery");
+const request = require("request");
+const url = require("url");
+const opentracing = require("opentracing");
+
+// go-swagger treats handles/expects arrays in the query string to be a string of comma joined values
+// so...do that thing. It's worth noting that this has lots of issues ("what if my values have commas in them?")
+// but that's an issue with go-swagger
+function serializeQueryString(data) {
+  if (Array.isArray(data)) {
+    return data.join(",");
+  }
+  return data;
+}
+
+module.exports = class SwaggerTest {
+
+  address;
+  timeout;
+
+  constructor(options) {
+    options = options || {};
+
+    if (options.discovery) {
+      this.address = discovery("swagger-test", "default").url();
+    } else if (options.address) {
+      this.address = options.address;
+    } else {
+      throw new Error("Cannot initialize swagger-test without discovery or address");
+    }
+    if (options.timeout) {
+      this.timeout = options.timeout
+    }
+  }
+
+  
+  deleteBook = (id, options, cb) => {
+    const params = {
+      
+        "id": id,
+      
+    };
+
+    if (!cb && typeof options === "function") {
+      cb = options;
+      options = undefined;
+    }
+
+    if (!options) {
+      options = {};
+    }
+
+    const timeout = options.timeout || this.timeout;
+    const span = options.span;
+
+    const headers = {
+
+    }
```

Comment:
missing semicolon (maybe we should add "generated code passes lint" to the wishlist?)